R/centrality.R

```diff
@@ -496,8 +498,21 @@ arpack_defaults <- list(
 #' }
 #' @family arpack
 #' @export
-arpack <- function(func, extra = NULL, sym = FALSE, options = arpack_defaults,
+arpack <- function(func, extra = NULL, sym = FALSE, options = arpack_defaults(),
                    env = parent.frame(), complex = !sym) {
+
+  eval_try <- rlang::eval_tidy(options)
```

Comment:
Why `eval_tidy()`? We should be able to do this with standard evaluation, forcing should be enough?

Reply:
do you have any example I could look at?